dev/reference/tools/tidb-binlog/reparo.md

```diff
@@ -81,6 +83,11 @@ log-level = "info"
 # If it is set to "mysql", you need to configure `host`, `port`, `user` and `password` in [dest-db].
 dest-type = "mysql"
 
+# Safe-mode configuration
+# Value: "true"/"false" ("false" by default)
+# If it is set to "true", Reparo splits the `UPDATE` statement into a `DELETE` statement plus a `REPLACE` statement.
+safe-mode = "false"
```

Comment:
@TomShawn @juliezhang1112 IMO, `safe-mode = false` is exactly what's in the configuration file. So we don't need `""` here. @ericsyh Is that right?

```suggestion
safe-mode = false
```